vault/external_tests/sealmigration/testshared.go

```diff
@@ -120,6 +124,58 @@ func ParamTestSealMigrationTransitToShamir_Post14(t *testing.T, logger hclog.Log
 	runShamir(t, logger, storage, basePort, rootToken, recoveryKeys)
 }
 
+func ParamTestSealMigrationTransitToShamir_Recovery(t *testing.T, logger hclog.Logger, storage teststorage.ReusableStorage, basePort int) {
+	// Create the transit server.
+	tss := sealhelper.NewTransitSealServer(t, 0)
+	defer func() {
+		if tss != nil {
+			tss.Cleanup()
+		}
+	}()
+	sealKeyName := "transit-seal-key-1"
+	tss.MakeKey(t, sealKeyName)
+
+	// Initialize the backend with transit.
+	cluster, opts := InitializeTransit(t, logger, storage, basePort, tss, sealKeyName)
+	rootToken, recoveryKeys := cluster.RootToken, cluster.RecoveryKeys
+
+	// Disable the transit seal, forcing recovery
+	tss.Cleanup()
```

Comment:
Feels like we should be doing `tss = nil` here, and removing the tss.Cleanup below.

Reply:
We need to shut down Transit so that it's unreachable and can't unseal without the recovery key method.  nilling it leaves the transit vault up with its port open right?

Reply:
I mean after calling Cleanup, assign nil.

Reply:
ahh